apps/signing/serializers.py

```diff
@@ -16,6 +16,7 @@ class FileUploadSerializer(serializers.ModelSerializer):
     passed_review = serializers.SerializerMethodField('get_passed_review')
     processed = serializers.BooleanField(source='processed')
     reviewed = serializers.SerializerMethodField('get_reviewed')
+    valid = serializers.BooleanField(source='passed_all_validations')
```

Comment:
It's a bit weird that the `FileUpload` has a `valid` but I'm thinking of this `valid` as "will it proceed to review". Which isn't the same as a `FileUpload`'s `valid` which is "were there any errors during validation (regardless of auto-signing validations)".

So it could be renamed but I think the name makes sense in terms of the API and we're trying to hide the thing it's wrapping with this API so I think it's fine?